### DISCUSSION TITLE
LEVEL SET  model running question

### DISCUSSION THREAD

**Discussion by LinKaiyi-design** [firemodels/fds#12661] [Q&A]
Could you please tell me the highest accuracy level for the LEVEL SET model in FDS that is currently available? I encountered a program interruption issue when running the FDS file with the LS model on HPC. I'm not sure if it's because my mesh resolution is too high.

LEVEL_SET_TEST.fds
&HEAD CHID='LEVEL SET TEST'/
&TIME T_END=3600.0/
&DUMP DT_RESTART=300.0, DT_DEVC=0.02,DT_SL3D=10,NFRAMES=18000/
&MISC TMPA= 23,HUMIDITY= 41,LEVEL_SET_MODE=4,/
&WIND speed=2,DIRECTION=130 /
&REAC FUEL='CELLULOSE', C=6, H=10, O=5, SOOT_YIELD=0.01, HEAT_OF_COMBUSTION=16400. /
&MESH MULT_ID='mesh', IJK=125,125,8, XB=0.0,125,0,125,0.0,8.0/
&MULT ID='mesh', DX=125.0, DY=125.0, DZ=8, I_UPPER=14, J_UPPER=14, K_UPPER=1 /
&VENT MB='XMAX', SURF_ID='OPEN' /
&VENT MB='XMIN', SURF_ID='MIRROR' /
&VENT MB='YMIN', SURF_ID='MIRROR' /
&VENT MB='YMAX', SURF_ID='MIRROR' /
&VENT MB='ZMAX', SURF_ID='MIRROR' /

&SURF ID='Barren',RGB=133,153,156/

&DEVC ID='tower_2m', XYZ=1100.0,885.0,2.0,  QUANTITY='U-VELOCITY'/ 
&DEVC ID='tower_2m', XYZ=1100.0,885.0,2.0, QUANTITY='V-VELOCITY'/ 
&DEVC ID='tower_2m', XYZ=1100.0,885.0,2.0, QUANTITY='W-VELOCITY'/ 
&DEVC ID='tower_6m', XYZ=1100.0,885.0,2.0,  QUANTITY='U-VELOCITY'/ 
&DEVC ID='tower_6m', XYZ=1100.0,885.0,2.0, QUANTITY='V-VELOCITY'/ 
&DEVC ID='tower_6m', XYZ=1100.0,885.0,2.0, QUANTITY='W-VELOCITY'/ 
&CATF OTHER_FILES='l2g_vegetation.txt' /

**Comment by mcgratta** [comment]
4

**Reply by LinKaiyi-design** [reply]
I mean the highest grid resolution in LEVEL SET model.Because I use the 5m-grid cells, the model can runs normally, but when using a 1m grid resolution, the program interrupts.

**Comment by mcgratta** [comment]
There are no limits in FDS, but I would guess that your computer has run out of memory.

**Reply by LinKaiyi-design** [reply]
Thank you! Dear Mr. Kevin,Could you please provide me with your recommendations regarding the optimal number of mesh cells for each MESH?  The configuration of my HPC platform is as follows：
900 CPU nodes: 2 * Intel Xeon 6258R processors, 192GB of memory (56 cores per node).

**Comment by marcosvanella** [comment]
Have you set your stack memory to unlimited?
In your ~/.bash_profile startup file add:

`ulimit -s unlimited`

**Reply by LinKaiyi-design** [reply]
No，thank you very much! I will try to add this in the file!
